Dockerfile

```diff
@@ -1,12 +1,17 @@
-FROM quay.io/azavea/scala:2.10.6
+FROM openjdk:7-jre
```

Comment:
If all this really needs is the JDK, is there an alpine version that we should give a shot since that would presumably be way smaller?